backend/inspirehep/mailing/cli.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+import click
+import structlog
+from flask.cli import with_appcontext
+
+from .api.jobs import (
+    get_jobs_from_last_week,
+    get_jobs_weekly_html_content,
+    send_jobs_weekly_campaign,
+)
+
+
+@click.group()
+def mailing():
+    """Command to handle mailing."""
+
+
+@mailing.command(help="Sends a campaign with the INSPIRE jobs posted last week.")
+@click.option("-t", "--test", help="Send test campaign.", is_flag=True)
+@click.option(
+    "-e",
+    "--email",
+    "emails",
+    help="Where to send the test email, ``--test`` is required.",
+    multiple=True,
+)
+@with_appcontext
+def send_weekly_jobs(test, emails):
+    if emails and not test:
```

Comment:
And what will happen if --test will be set and --emails wont? User might thing that only test campaign will be sent. Why you require both parameters if you are not doing anything with test? Isn't better to just call `--email` parameter `--test-emails` and remove --test?